pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -204,41 +164,37 @@ func (uc *UpgradeCluster) getClusterNodeStatus(subscriptionID uuid.UUID, az armh
 		}
 
 		for _, vm := range vmListPage.Values() {
-			vmOrchestratorTypeAndVersion := uc.getClusterNodeVersion(kubeClient, *vm.Name, vm.Tags)
-			if vmOrchestratorTypeAndVersion == "" {
+			// Windows VMs contain a substring of the name suffix
```

Comment:
What is the Windows-specific implications of the line(s) below?

Reply:
`strings.Contains(*(vm.Name), uc.NameSuffix[:4]+"k8s"` is checking for VM naming specific to windows instead of looking for the whole name suffix